services/token_reject.proto

```diff
@@ -0,0 +1,68 @@
+syntax = "proto3";
+
+package proto;
+
+/*-
+ * ‌
+ * Hedera Network Services Protobuf
+ * ​
+ * Copyright (C) 2018 - 2024 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+option java_package = "com.hederahashgraph.api.proto.java";
+// <<<pbj.java_package = "com.hedera.hapi.node.token">>> This comment is special code for setting PBJ Compiler java package
+option java_multiple_files = true;
+
+import "basic_types.proto";
+
+/**
+ * Reject undesired token(s).<br/>
+ * Transfer one or more token balances held by the requesting account to the treasury for each
+ * token type.<br/>
+ * Each transfer SHALL be one of the following
+ * - A single non-fungible/unique token.
+ * - The full balance held for a fungible/common token type.
+ *
+ * A single tokenReject transaction SHALL support a maximum of 10 transfers.
+ */
+message TokenRejectTransactionBody {
+  /**
+   * A list of one or more token rejections.<br/>
+   * On success each rejected token serial number or balance SHALL be transferred from
+   * the requesting account to the treasury account for that token type.<br/>
+   * After rejection the requesting account will still be associated with the token and
+   * if token dissociation is needed then a separate TokenDissociate transaction can be used to achieve it.
+   */
+  repeated TokenReference rejections = 1;
+}
+
+/**
+ * A union token identifier.<br/>
+ * Identify a fungible/common token type, or a single non-fungible/unique token serial.
```

Comment:
This will read better in documentation with a slight change:
```suggestion
 * A union token identifier.
 *
 * Identify a fungible/common token type, or a single non-fungible/unique token serial.
```